pkg/didcomm/protocol/exchange/exchange.go

```diff
@@ -10,15 +10,60 @@ import (
 	"encoding/base64"
 	"encoding/json"
 
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/protocol/decorator"
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/transport"
+	"github.com/hyperledger/aries-framework-go/pkg/doc/did"
 	"github.com/pkg/errors"
 )
 
+const (
+	connectionInvite   = "did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/connections/1.0/invitation"
```

Comment:
Suggest we pull the did:sov:BzCbsNYhMrjHiqZDTUASHg Aries Community identifier out into its own constant. It would also be an idea to put that constant into a common package.

Reply:
Moved to a separate package https://github.com/hyperledger/aries-framework-go/compare/c940bd1b350c2339769d648b80dc9fc401207265..1446f2c0373b23d55b26c068d9e9989fca4b8681#diff-6aefc64b8fb45a2877cdf8e40aa42116R10